frameworks/TypeScript/nest/src/app.service.ts

```diff
@@ -0,0 +1,8 @@
+import { Injectable } from '@nestjs/common';
```

Comment:
We may be able to do away with all the boilerplate code and just leave the `BenchController`